src/components/cron.ts

```diff
@@ -50,6 +52,12 @@ export const createOfficeStatusCron = (client: Client): CronJob =>
     } else {
       throw 'Bad channel type';
     }
+
+    if (response['status'] == 1) {
+      // Send all the users with the "Office Ping" role a DM:
+      // Get all users with "Office Ping" role
+      await alertUsers();
+    }
```

Comment:
It looks like this would DM the users every 10 minutes while the office is continuously open. I believe that we only want the users to be DMed once, within the first 10 minutes of the office opening.